apps/client/src/pages/DevHomePage.js

```diff
@@ -1,37 +1,41 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Link } from "react-router-dom";
 
 import { topics } from "../config";
 import { routes, geturl } from "../routes";
 
 import Layout from "../components/Layouts/DefaultLayout";
+import Context from "../context";
 
-const DevHomePage = () => (
-  <Layout heading="Hi Dev!">
-    <p>Welcome to CHAPPIE 1.0</p>
-    <table>
-      <thead>
-        <tr>
-          <th>Flow</th>
-          <th>slug</th>
-        </tr>
-      </thead>
-      <tbody>
-        {topics
-          .sort((a) => !!a.sttrFile && -1) // make sure sttr flow is on top
-          .map(({ slug, sttrFile, redirectToOlo }) => (
-            <tr key={slug}>
-              <td>
-                {redirectToOlo ? "Redirect" : sttrFile ? "Checker" : "OLO"}
-              </td>
-              <td>
-                <Link to={geturl(routes.intro, { slug })}>{slug}</Link>
-              </td>
-            </tr>
-          ))}
-      </tbody>
-    </table>
-  </Layout>
-);
-
+const DevHomePage = () => {
+  const context = useContext(Context);
+  context.trackEvents = false;
```

Comment:
I don't understand why this would be done if you get to DevHomePage. What if you don't visit this page? What if you visit this page half-way in your session?